tensorflow/core/distributed_runtime/master.cc

```diff
@@ -147,16 +150,21 @@ class DeviceFinder {
 
   static void GetRemoteWorkers(
       const protobuf::RepeatedPtrField<string>& device_filters, MasterEnv* env,
-      WorkerCacheInterface* worker_cache, std::vector<string>* workers) {
-    DeviceFinder finder(device_filters, env, worker_cache);
+      WorkerCacheInterface* worker_cache, std::vector<string>* workers,
+      uint64 device_finder_timeout_in_micros) {
+    DeviceFinder finder(device_filters, env, worker_cache,
+                        device_finder_timeout_in_micros);
     *workers = finder.targets_;
   }
 
  private:
   explicit DeviceFinder(
       const protobuf::RepeatedPtrField<string>& device_filters, MasterEnv* env,
-      WorkerCacheInterface* worker_cache)
+      WorkerCacheInterface* worker_cache, uint64 timeout_in_micros)
       : env_(env), worker_cache_(worker_cache) {
+    if (timeout_in_micros > 0) {
+      DeviceFinder::timeout_in_micros_ = timeout_in_micros;
```

Comment:
Prefer to use class member initializer (i.e., `timeout_in_micros_(timeout_in_micros)`) since the if condition does not seem to be necessary.

Reply:
Done.